plaidml/op/CPPLINT.cfg

```diff
@@ -0,0 +1 @@
+filter=-whitespace/line_length
```

Comment:
Hmm... Does FileCheck have troubles with NOLINT directives? (Meaning: append `// NOLINT[whitespace/line_length]` to lines that are too long). If that works, it's preferable to turning off that part of the linter for all of `plaidml/op`.

Reply:
I'll try that out; I'd just been following what had been done in the edsl testing

Reply:
Yep, this is correct for lit tests